system/settings/settings.xml

```diff
@@ -3724,6 +3724,13 @@
           <control type="slider" format="integer" />
         </setting>
       </group>
+      <group id="2" label="39180">
+        <setting id="osd.showcommbreaknotification" type="boolean" label="39181" help="39182">
+          <level>2</level>
```

Comment:
Is this edge case setting really level 2?

Reply:
I am not sure either. I see advancedsettings.xml as something that should be avoided as much as possible, ultimately the place for changing settings that should not be changed unless the user really knows what he's doing (cache level, cache mode, etc). This setting avoids the display of a notification...Kodi is guaranteed to work well with or without this setting.
I see it as some gui setting at advanced or expert levels (more due to the fact it refers to EDL - which is not some main stream functionality - rather than what it actually does).
Level 2 is advanced, I can move it one level up if needed